src/amp-shadow-babel.js

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
I think you no longer need this file. Let me know and we can delete the other one.